x/ccv/provider/types/keys.go

```diff
@@ -685,6 +685,11 @@ func ConsumerIdToPhaseKey(consumerId string) []byte {
 	return StringIdWithLenKey(mustGetKeyPrefix(ConsumerIdToPhaseKeyName), consumerId)
 }
 
+// ConsumerIdToPhaseKeyPrefix returns the key prefix used to iterate over all the consumer ids and their phases.
```

Comment:
I got it

Reply:
See @mpoke comment [here](https://github.com/cosmos/interchain-security/pull/2117#discussion_r1706771493):
> `mustGetKeyPrefix(ConsumerIdToRegistrationRecordKeyName)`. The idea is to have only one method calling mustGetKeyPrefix for a given key and for all these methods to be part of the UTs so that we make sure the panic in mustGetKeyPrefix is not triggered.